deployments/docker/docker-compose.yml

```diff
@@ -0,0 +1,93 @@
+        VERIFICATION_DEPLOY_PREFIX: ${VERIFICATION_DEPLOY_PREFIX:?Please define a value for the environment variable}
+        VTS_DEPLOY_PREFIX: ${VTS_DEPLOY_PREFIX:?Please define a value for the environment variable}
+        VTS_PROVISIONING_LOCAL_IP_ADDRESS: ${VTS_PROVISIONING_LOCAL_IP_ADDRESS:?Please define a value for the environment variable}
+        VTS_VERIFICATION_LOCAL_IP_ADDRESS: ${VTS_VERIFICATION_LOCAL_IP_ADDRESS:?Please define a value for the environment variable}
+        VTS_PROVISIONING_LOCAL_IP_ADDRESS_PORT: ${VTS_PROVISIONING_LOCAL_IP_ADDRESS_PORT:?Please define a value for the environment variable}
+        VTS_VERIFICATION_LOCAL_IP_ADDRESS_PORT: ${VTS_VERIFICATION_LOCAL_IP_ADDRESS_PORT:?Please define a value for the environment variable}
+        BIN_DIR: ${BIN_DIR:?Please define a value for the environment variable}
+        LOG_DIR: ${LOG_DIR:?Please define a value for the environment variable}
+        PLUGIN_DIR: ${PLUGIN_DIR:?Please define a value for the environment variable}
+        INPUT_FILE_DIR: ${INPUT_FILE_DIR:?Please define a value for the environment variable}
+    ports:
+      - 8080:8080
+    depends_on:
+      - vts
+    networks:
+      - vts-verif
+
+
+  vts:
+    build:
+      context: ../../
+      dockerfile: ./deployments/docker/Dockerfile
+      target: vts-run
+      args:
+        PROVISIONING_DEPLOY_PREFIX: ${PROVISIONING_DEPLOY_PREFIX:?Please define a value for the environment variable}
+        VERIFICATION_DEPLOY_PREFIX: ${VERIFICATION_DEPLOY_PREFIX:?Please define a value for the environment variable}
+        VTS_DEPLOY_PREFIX: ${VTS_DEPLOY_PREFIX:?Please define a value for the environment variable}
+        VTS_PROVISIONING_LOCAL_IP_ADDRESS: ${VTS_PROVISIONING_LOCAL_IP_ADDRESS:?Please define a value for the environment variable}
+        VTS_VERIFICATION_LOCAL_IP_ADDRESS: ${VTS_VERIFICATION_LOCAL_IP_ADDRESS:?Please define a value for the environment variable}
+        VTS_PROVISIONING_LOCAL_IP_ADDRESS_PORT: ${VTS_PROVISIONING_LOCAL_IP_ADDRESS_PORT:?Please define a value for the environment variable}
+        VTS_VERIFICATION_LOCAL_IP_ADDRESS_PORT: ${VTS_VERIFICATION_LOCAL_IP_ADDRESS_PORT:?Please define a value for the environment variable}
+        BIN_DIR: ${BIN_DIR:?Please define a value for the environment variable}
+        LOG_DIR: ${LOG_DIR:?Please define a value for the environment variable}
+        PLUGIN_DIR: ${PLUGIN_DIR:?Please define a value for the environment variable}
+        INPUT_FILE_DIR: ${INPUT_FILE_DIR:?Please define a value for the environment variable}
+    networks:
+      vts-prov:
+        ipv4_address: ${VTS_PROVISIONING_LOCAL_IP_ADDRESS:?Please define a value for the environment variable}
+      vts-verif:
+        ipv4_address: ${VTS_VERIFICATION_LOCAL_IP_ADDRESS:?Please define a value for the environment variable}
+
+networks:
+  default:
+    external: true
+    name: none
+
+
+  vts-prov:

```

Comment:
just a thought (ignore if you disagree): should this be called provisioning-network?
And similarly the one below.

Reply:
Yes I think provisioning-network is more clear (and the one below too)